pandas/core/generic.py

```diff
@@ -1913,12 +1914,12 @@ def __len__(self) -> int:
         """Returns length of info axis"""
         return len(self._info_axis)
 
-    def __contains__(self, key):
+    def __contains__(self, key) -> bool_t:
         """True if the key is in the info axis"""
         return key in self._info_axis
 
     @property
-    def empty(self):
+    def empty(self) -> bool_t:
```

Comment:
should we be generally standarizing on using ``bool_t`` rather than ``bool``?

Reply:
IMO probably not necessary. probably only a few modules where this pattern is needed.